submissions/kikinovk/HTML_CSS_Popup/css/style.css

```diff
@@ -0,0 +1,265 @@
+* {
+  margin: 0;
+  padding: 0;
+  outline:none;
+}
+html {
+  height: 100%;
+}
+body {
+  display: grid;
+  min-height: 100%;
+  grid-template-rows: auto 1fr;
+  font-family: sans-serif;
+  font-size: 16px;
+  color: #666666;
+}
+
+/* --Navigation-- */
+
+.nav {
+  padding: 10px;
+}
+.nav__menu {
+  display: grid;
+  column-gap: 30px;
+  grid-template-columns: repeat(5, max-content);
+  justify-content: end;
+}
+.menu__item {
+  display: block;
+  line-height: 1.5em;
+}
+.menu__icon {
+  display: block;
+  width: 1.5em;
+  height: 1.5em;
+  background-size: contain;
+}
+.menu__link {  
+  text-decoration: none;
+  color: #666666;
+}
+.menu__link:hover, .menu__link:focus{
+  color: #0033cc;
+  text-decoration: underline;
+}
+.menu__link:focus > .icon--bell {
+  background-image: url(../image/bell-hover.png);
```

Comment:
Changing different images by using `background-image` on each hover or focus can drastically decrease the performance of your styles. Better to use SVG icons here or some font icons and just toggle `fill` or `stroke` properties (for the SVGs) or color (for the font icons) on hover/focus/default states.
Let's leave these icons. But don't forget to use the SVG or font icons in the future, please